build.sbt

```diff
@@ -82,6 +82,17 @@ lazy val commonSettings = Release.publishSettings ++ Seq (
   )
 )
 
+lazy val sunriseModuleDependencies = Seq (
```

Comment:
You're right, in this case it makes no sense to put it for a particular set of dependencies anymore. I've pulled it up.